libs/geoservices/covid/src/lib/pages/maps/components/time-map/time-map.component.html

```diff
@@ -0,0 +1,50 @@
+<section>
+  <div id="map-section">
+    <tamu-gisc-drawer [width]="'25rem'" [position]="'left'">
+      <div class="content">
+        <h3>Map Controls</h3>
+        <label class="switch">
+          <input type="checkbox" id="csToggle" (click)="stateToggle(this)">
+          <span class="slider round"></span>
+        </label>
```

Comment:
Label should not wrap the input element.
```html
<input type="checkbox" name="county-state-toggle">
<label for="county-state-toggle>Counties/State</label>
```

Reply:
used this checkbox setup and removed the old switch and it's css